airtable/plugin.py

```diff
@@ -405,6 +406,69 @@ def airtable_record_create(
     )
 
 
+@purpose("Create multiple records in batch.")
+def airtable_records_create(
+    base_id: AirtableBaseID,
+    table_id: AirtableTableID,
+    records: Sequence[dict[str, Any]],
+    typecast: bool = True,
+) -> list[AirtableRecord]:
+    """
+    Create multiple records using the Airtable `create records` API call with a POST.
+    Takes a list of field dictionaries and creates a record for each one.
+
+    Prefer to use this over `airtable_record_create` since this takes batches of records
```

Comment:
Reminder: delete this suggestion once you add `@deprecated`.

Reply:
Done.